### PR TITLE
fix serial event

### PR DESCRIPTION
**Pull request by pelikhan**
@jamesadevine the event does not trigger after a few occurences

### CODE REVIEW

**Comment by jamesadevine** [comment]
Remember to drain the buffer! :wink:

File: libs/serial/enums.d.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
     declare const enum SerialEvent {
     //% block="data received"
-    DataReceived = 3,  // CODAL_SERIAL_EVT_RX_FULL
+    DataReceived = 4,  // CODAL_SERIAL_EVT_DATA_RECEIVED
```
Comment (jamesadevine) [inline libs/serial/enums.d.ts]: Swap to use define now dal.ts has been refreshed?

Comment (pelikhan) [reply libs/serial/enums.d.ts]: This is the compiled definition built from serial.cpp

Comment (jamesadevine) [reply libs/serial/enums.d.ts]: Oh right, sorry!

**Pull request merged**